src/taco-cli/cli/utils/settings.ts

```diff
@@ -102,6 +102,30 @@ class Settings {
         });
     }
 
+    /**
+     * Apply functions to all local and remote platforms.
+     * Local platforms are batched into one list because Cordova does not cope with multiple concurrent invocations
+     * Remote platforms are done individually (but concurrently with each other and local platforms) since they may
+     * go to completely different servers.
+     */
+    public static operateOnPlatforms(platforms: Settings.IPlatformWithLocation[],
```

Comment:
It seems to be highly related to the Platform concept, so why not PlatformUtilities? or PlatformOperations?